### DISCUSSION TITLE
Ayuda con el Kernel en Github

### DISCUSSION THREAD

**Discussion by Edgarsantome** [LaunchX-InnovaccionVirtual/CursoIntroPython#41] [Q&A]
Hola buenos días
En la parte de Github cuando quiero ejecutar la parte del código, no me permite seleccionar ningún kernel, y me dice que la funcionalidad no es permitida como ya se comentó en un video y adicional me dice que si quiero hacer un clone a un repositorio local, pero aún así  no hace esa funcionalidad ya busqué información, pero no doy, Si hago los ejercicios  en el editor a nivel  local no tengo ningún problema. Alguien que me pueda ayudar, por favor

**Comment by EnriquePH77** [comment]
Creo que es porque el editor online de giuthub no tiene esa funcionalidad y necesitas hacerlo a nivel local.

**Comment by Paulina-Ceciliano** [comment]
Primero debes instalar python en tu computadora, después visual studio code, después instalas la extensión de Jupyter y después en la esquina superior derecha te va a apareces esto
![image](https://user-images.githubusercontent.com/88915301/153244509-6524666b-9790-45a8-8c31-5e5025e13349.png)
ahí es donde te da a seleccionar un kernel, lo seleccionas y listo. 
Pero si antes de seleccionar un kernel intentas darle play a una celda de código te va a decir que necesita instalar un kernel, le das que si y se instalan unos paquetes y se selecciona el kernel.

**Reply by AlejandroMontoyaT** [reply]
Te faltó la extensión de python en VSC y después Júpiter

**Comment by PARZANTONIO** [comment]
Me sucedió de manera local, por haber instalado python desde el Microsoft Store; No me aparecía la opción de seleccionar el kernel y por lo tanto el Run tampoco.
Solucioné, desinstalando la versión de Microsoft store y descargando e instalando la versión desde https://www.python.org/downloads/
